### PR TITLE
add system fiability integration skip tool process in message welcome…

### PR DESCRIPTION
**Pull request by ThomasDev28**
… message

add icon tools ui && fix fiability on message

delete env

### CODE REVIEW

File: src/Components/Conversation.module.css [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  font-family: "Nunito Sans",-apple-system,".SFNSText-Regular","San Francisco",BlinkMacSystemFont,"Segoe UI","Helvetica Neue",Helvetica,Arial,sans-serif;
   position: relative;
   height: 100%;
   width: 100%;
   overflow: hidden;
   background-color: rgba(255, 255, 255, 0.8);
   backdrop-filter: blur(10px);
-  min-width: 300px;
+  /* min-width: 300px; */
+  min-width: 600px;
```
Comment (xonlly) [inline src/Components/Conversation.module.css]: 300px

File: package.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
     "build:rl": "rollup -c --bundleConfigAsCjs",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
-    "prepublish": "yarn build",
+    "": "yarn build",
```
Comment (xonlly) [inline package.json]: prepublish

File: src/Components/Conversation.tsx [hunk 3]
```diff
@@ -144,7 +186,15 @@ export const Conversation: React.FC<IProps> = ({
         });
       },
       onMessage: (message) => {
-        setIsPending(false);
+        if (displayTools) {
+          setIsPending(false);
+        }
+
+        if (message.trim().length > 0 && !displayTools) {
+          console.log("message.trim().length > 0 && !displayTools", message);
```
Comment (xonlly) [inline src/Components/Conversation.tsx]:
```suggestion
```

File: src/Components/Conversation.tsx [hunk 4]
```diff
@@ -164,12 +215,34 @@ export const Conversation: React.FC<IProps> = ({
   };
 
   const handleResetConversation = () => {
+    console.log("handleResetConversation");
```
Comment (xonlly) [inline src/Components/Conversation.tsx:217]:
```suggestion
```

File: src/Components/Conversation.tsx [hunk 5]
```diff
@@ -248,8 +327,89 @@ export const Conversation: React.FC<IProps> = ({
...
+                            const send =
+                              m.fiability === fiability_key
+                                ? "DEFAULT"
+                                : fiability_key;
+
+                            handleFiabilityMessage(m, send);
+                          }}
+                          className={styles["btn-container-fiability"]}
+                        >
+                          {fiability_key === "BAD" ? (
```
Comment (xonlly) [inline src/Components/Conversation.tsx]: Why not use, const for that ?
```suggestion
                          {fiability_key === FiabilityEnum.BAD ? (
```

File: assets/google.png
Comment (xonlly) [inline assets/google.png:1]: Please resize this picture to 28px by 28

**Review by xonlly — CHANGES_REQUESTED** [review]

**Comment by xonlly** [comment]
Please update Markdown with new props

File: src/config.ts [hunk 6]
```diff
@@ -3,7 +3,7 @@ import { MarkdownPreviewProps } from "@uiw/react-markdown-preview";
 /**
  * The URL for the API (default: https://api.devana.ai/).
  */
-export const API_URL = `https://api.devana.ai/`;
```
Comment (Slashgear) [inline src/config.ts]: Issue:
Are we sure we want to use the http version ?
We will need to use https

**Review by Slashgear — CHANGES_REQUESTED** [review]

**Review by Slashgear — APPROVED** [review]

**Review by xonlly — APPROVED** [review]
🌈

**Pull request merged**